tests/CliTestCase/testLoadtasksLoadsDoctrineStyleTasksFromTheSpecifiedDirectory/should-be-ignored.php

```diff
@@ -1,9 +1,9 @@
 <?php
 /**
  * Fixture for Doctrine_Cli_TestCase
- * 
+ *
  * @author Dan Bettles <danbettles@yahoo.co.uk>
  */
 
-This file should be ignored because its filename starts with a lowercase letter.  You shouldn't, therefore, see any
-syntax errors when running Doctrine_Cli_TestCase.
\ No newline at end of file
+// This file should be ignored because its filename starts with a lowercase letter.
+throw new Exception('This should not happen');
```

Comment:
php-cs-fixer also complained about this file since it has syntax errors.

I figured that making it throw an exception rather than having syntax errors should suffice for this error condition.